packages/server-wallet/src/wallet/index.ts

```diff
@@ -99,8 +100,21 @@ export class Wallet implements WalletInterface {
   async joinChannel(_args: JoinChannelParams): Result {
     throw 'Unimplemented';
   }
-  async updateChannel(_args: UpdateChannelParams): Result {
-    throw 'Unimplemented';
+  async updateChannel({channelId, appData, allocations}: UpdateChannelParams): Result {
+    return Channel.transaction(async tx => {
+      const channel = await Store.getChannel(channelId, undefined);
```

Comment:
See [this comment](https://github.com/statechannels/statechannels/pull/2368/files#r461213698). We could have:
```ts
const query = await Store.getChannel(channelId, undefined);
if (isNone(query)) return UpdateChannelError.channelNotFound(channelId);
const {value: channel} = query;
```

Reply:
Fixed in 572a3ec.